src/PIL/PcfFontFile.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, fp, charset_encoding="iso8859-1"):
 
         for ch, ix in enumerate(encoding):
             if ix is not None:
-                x, y, l, r, w, a, d, f = metrics[ix]
+                x, y, l, r, w, a, d, f = metrics[ix]  # noqa: E741
```

Comment:
Please can you check what `l` represents? Is there a better name?
(And below)

Reply:
https://github.com/python-pillow/Pillow/blob/583dd3b4245649ddd367c2dcbbf2bf9fb7835017/src/PIL/PcfFontFile.py#L168